cbfeeds/feed.py

```diff
@@ -87,46 +95,67 @@ def validate(self, pedantic=False, serialized_data=None):
 class CbFeedInfo(object):
     def __init__(self, **kwargs):
         # these fields are required in every feed descriptor
-        self.required = ["name", "display_name", "version",
+        self.required = ["name", "display_name",
                          "summary", "tech_data", "provider_url"]
+        self.optional = ["category", "icon", "version", "icon_small"]
```

Comment:
Can you add a description of "category" to the README, and what the allowed values are?